marlowe-runtime/cli/Language/Marlowe/Runtime/CLI/Command/Create.hs

```diff
@@ -0,0 +1,84 @@
+module Language.Marlowe.Runtime.CLI.Command.Create
+  where
+
+import Data.Map (Map)
+import qualified Data.Map as Map
+import Data.String (fromString)
+import Language.Marlowe (POSIXTime(POSIXTime))
+import Language.Marlowe.Runtime.CLI.Command.Tx (TxCommand, txCommandParser)
+import Language.Marlowe.Runtime.CLI.Monad (CLI)
+import Language.Marlowe.Runtime.CLI.Option (keyValueOption, marloweVersionParser, parseAddress)
+import Language.Marlowe.Runtime.ChainSync.Api (Address, TokenName(..))
+import Language.Marlowe.Runtime.Core.Api (SomeMarloweVersion)
+import Options.Applicative
+import Text.Read (readMaybe)
+
+data CreateCommand = CreateCommand
```

Comment:
For all of the commands, but especially for `create`, I think we should allow arbitrary transaction metadata to be included in the transaction: i.e., any metadata key (not just 1564). For example, it is very common to use the CIP-20 message metadata (key 674) in transactions, for instance.

Reply:
I forgot about this, I'll add an argument for this.